### ISSUE TITLE
The "Awareness" page has no text alternatives for non-text content

### ISSUE THREAD

**Issue by mah-c** [mah-c/wk-6-mah-c-1-glitch-grabbers-team-repo#6]
The Awareness page has no text alternatives for non-text contents such as images etc. This would be helpful to users who are using screen readers to access the website.

Steps to reproduce
1. Navigate to the awareness page
2. click ctrl+u
3. Observe the source code for <img> tags

Expected behavior
The non text content such images should have "alt" to describe the image to screen readers making it accessible to users with disability (visually impaired)

Actual behavior
There are no "alt" texts for non-text content.

Screenshot

<img width="991" height="443" alt="Image" src="https://github.com/user-attachments/assets/af540015-9851-4ef3-a4e9-6d20d00b1255" />
